### PR TITLE
Fixed syntax errors on 4.4

### PR DESCRIPTION
**Pull request by Nyholm**
I ran the [new parser](https://github.com/symfony/symfony-docs/pull/15257) on 4.4. There are about 130 more syntax errors that I decided not to fix. Some are false positives (ie a code example where we only have a PHP array). Others show some limitations of the parser. Ie, we cannot automatically detect a code block with no language, we currently assume it is PHP but that is not always correct. 

This is the last fixes I'll do for 4.4.

### CODE REVIEW

File: mercure.rst [hunk 1]
```diff
@@ -645,13 +645,14 @@ During functional testing you can instead decorate the Hub::
     }
 
 HubStub decorates the default hub service so no updates are actually
-sent. Here is the HubStub implementation::
+sent. Here is the HubStub implementation:
+
+.. code-block:: yaml
```
Comment (Nyholm) [inline mercure.rst:650]: Should I revert this and instead focus on https://github.com/symfony-docs-tools/code-block-checker/issues/20

There are many more yaml/json/xml code blocks like this.

Comment (wouterj) [reply mercure.rst:650]: This is a bug, so please fix them :) (and not the checker)

File: service_container.rst [hunk 2]
```diff
@@ -344,15 +344,18 @@ you can type-hint the new ``SiteUpdateManager`` class and use it::
     // ...
     use App\Service\SiteUpdateManager;
```
Comment (wouterj) [inline service_container.rst]:
```suggestion
    use App\Service\SiteUpdateManager;
    // ...
```

Sorry, this is extremely nitpicking-me (and I'm probably the only idiot actively thinking about it), but the collapsed use statements now includes at least 1 `Symfony` namespace, which should be located after the `App` namespace (so the `// ...` is placed after the existing use statement).

**Review by wouterj — APPROVED** [review]
So many good fixes! :heart_eyes: (actually: 100% good fixes)

File: service_container.rst (hunk 2 above)
Comment (Nyholm) [reply service_container.rst]: Okey. Thank you

**Comment by Nyholm** [comment]
Thank you for the review. I've updated the non-php code blocks.

**Comment by javiereguiluz** [comment]
Tobias, you're on a roll! What an amazing stream of cool PRs you are contributing lately 😍  Thank you 🙇‍♂️

**Comment by Nyholm** [comment]
Im happy you are appreciating the work. 

Thank you for merging

**Pull request merged**